web/layout.go

```diff
@@ -37,7 +37,7 @@ type SubmenuItem struct {
 }
 
 var defaultLayoutData = LayoutData{
-	Title:     "Trento Console",
+	Title:     "Trento Console for SAP Applications",
```

Comment:
why this change? it is already long enough imho ;)

Reply:
I thought I saw it in some of the UI mocks but I can't seem to find it anymore. I'll change it!